lib/identity_doc_auth/acuant/request.rb

```diff
@@ -41,7 +41,15 @@ def fetch
         http_response = send_http_request
         return handle_invalid_response(http_response) unless http_response.success?
```

Comment:
> That is super strange. The tests pass locally without issue...

I think you need to remove this line. Mine fails locally, removing this line has the expected bhavior

```suggestion
```

Reply:
Wow, that's wild. My IDE's git integration actually screwed up on me. That's not there on my local so I missed it. It can happen but I don't think I've ever seen it personally. Thanks for spotting that.